.github/workflows/build.yml

```diff
@@ -30,3 +30,8 @@ jobs:
       run: dotnet restore
     - name: Run build
       run: dotnet fake build -t Test
+    - name: Run Fsharplint on itself
+      run: |
+        cd ./src/FSharpLint.Console
+        dotnet run --framework net5.0 lint ../FSharpLint.Core/FSharpLint.Core.fsproj
+        dotnet run --framework net5.0 lint ./FSharpLint.Console.fsproj
```

Comment:
@Mersho man, look at the scope of the commit msg where you added this change, do you think it's correct?